### ISSUE TITLE
fish: `base16-fish` script runs twice

### ISSUE THREAD

**Issue by donovanglover** [danth/stylix#527]
Noticed while finding the cause of https://github.com/danth/stylix/issues/526. Seems to be caused by using `interactiveShellInit` and `promptInit`.

**Comment by danth** [comment]
Yes, this will be because it appears in both the NixOS configuration and the Home Manager configuration when they're used together.

We could prevent it by setting an environment variable once the script has completed, then testing for that environment variable before trying to run it again.
